Pipfile

```diff
@@ -6,54 +6,54 @@ verify_ssl = true
 [dev-packages]
 Sphinx = "==2.1.2"
 black = "==19.3b0"
-coveralls = "==1.8.1"
+coveralls = "==1.8.2"
 django-extensions = "==2.2.1"
+doc8 = "==0.8.0"
 factory_boy = "==2.12.0"
 flake8 = "==3.7.8"
 ipython = "==7.7.0"
 isort = "==4.3.21"
+pytest = "==4.6.5"
 pytest-asyncio = "==0.10.0"
 pytest-cov = "==2.7.1"
 pytest-django = "==3.5.1"
 pytest-factoryboy = "==2.0.3"
 pytest-mock = "==1.10.4"
-pytest = "==4.6.4"
-sphinxcontrib-httpdomain = "==1.7.0"
-doc8 = "==0.8.0"
 remote-pdb = "==2.0.0"
+sphinxcontrib-httpdomain = "==1.7.0"
 
 [packages]
 sfdo-template-helpers = {editable = true,git = "https://github.com/SFDO-Tooling/sfdo-template-helpers.git",ref = "v0.9.0"}
-boto3 = "==1.9.197"
-channels-redis = "==2.4.0"
+Django = "==2.2.4"
+Pillow = "==6.1.0"
+Werkzeug = "==0.15.5"
+bleach = "==3.1.0"
+boto3 = "==1.9.200"
 channels = "==2.2.0"
+channels-redis = "==2.4.0"
 dj-database-url = "==0.5.0"
 django-allauth = "==0.39.1"
+django-filters = "==0.2.1"
 django-hashid-field = "==2.1.6"
 django-js-reverse = "==0.9.1"
 django-log-request-id = "==1.3.2"
 django-model-utils = "==3.2.0"
 django-parler = "==2.0"
 django-redis = "==4.10.0"
-django-rq-scheduler = "==1.1.3"
 django-rq = "==2.1.0"
+django-rq-scheduler = "==1.1.3"
 django-storages = "==1.7.1"
-djangorestframework = "==3.10.1"
+djangorestframework = "==3.10.2"
+github3-py = "==1.3.0"
 logfmt = "==0.4"
+markdown = "==3.1.1"
 newrelic = "==4.20.1.121"
 psycopg2-binary = "==2.8.3"
-sentry-sdk = "==0.10.2"
-whitenoise = "==4.1.3"
-Django = "==2.2.4"
-Pillow = "==6.1.0"
+purl = "==1.5"
 pyOpenSSL = "==19.0.0"
+sentry-sdk = "==0.10.2"
 service_identity = "==18.1.0"
-Werkzeug = "==0.15.5"
-github3-py = "==1.3.0"
-purl = "==1.5"
-django-filters = "==0.2.1"
-bleach = "==3.1.0"
-markdown = "==3.1.1"
+whitenoise = "==4.1.3"
```

Comment:
@wlonk Does order matter here? I alphabetized...

Reply:
Nope, order does not matter.